packs/feats/aldori-duelist-dedication.json

```diff
@@ -28,7 +28,28 @@
             "remaster": false,
             "title": "Pathfinder Lost Omens: World Guide"
         },
-        "rules": [],
+        "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "PF2E.Skill.Acrobatics",
+                        "value": "acrobatics"
+                    },
+                    {
+                        "label": "PF2E.Skill.Athletics",
+                        "value": "athletics"
+                    }
+                ],
+                "flag": "skill",
+                "key": "ChoiceSet"
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "upgrade",
+                "path": "system.skills.{item|flags.pf2e.rulesSelections.skill}.rank",
+                "value": "ternary(gte(@actor.system.skills.{item|flags.pf2e.rulesSelections.skill}.rank,1),2,1)"
```

Comment:
This feels a bit unstable. Maybe better to have 4 choices, only 2 of which get shown?
```json
{
    "adjustName": false,
    "key": "ChoiceSet",
    "choices": [{
        "label": "PF2E.Skill.Acrobatics",
        "value": {
            "slug": "acrobatics",
            "rank": 1
        },
        "predicate": [
            "skill:acrobatics:rank:0"
        ]
    }, {
        "label": "PF2E.Skill.Athletics",
        "value": {
            "slug": "athletics",
            "rank": 1
        },
        "predicate": [
            "skill:athletics:rank:0"
        ]
    }, {
        "label": "PF2E.Skill.Acrobatics",
        "value": {
            "slug": "acrobatics",
            "rank": 2
        },
        "predicate": [
            "skill:acrobatics:rank:1"
        ]
    }, {
        "label": "PF2E.Skill.Athletics",
        "value": {
            "slug": "acrobatics",
            "rank": 2
        },
        "predicate": [
            "skill:athletics:rank:1"
        ]
    }],
    "prompt": "PF2E.SpecificRule.Prompt.Skill"
}
```

Reply:
The ternary was working in testing various cases, but I was still uneasy about it. I like this more.